.mci.yml

```diff
@@ -145,15 +147,19 @@ cxx_driver_variables:
           git clone https://github.com/10gen/mongo-orchestration.git
           cd mongo-orchestration
           python setup.py install
+          cd ..
```

Comment:
note that i am moving this up so that mongo-orchestrations server.pid file will be in the main test directory so that mongo-orchestration stop works cleanly